### PR TITLE
Added broker optional parameters into configuration section of overview document 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: doc/MSB.md [hunk 1]
```diff
@@ -326,6 +326,17 @@ See for more [detail](https://www.rabbitmq.com/tutorials/amqp-concepts.html).
...
 `consumerThreadPoolQueueCapacity` – maximum number of requests waiting in FIFO queue to be processed by consumer thread pool. Should be positive integer or -1. Value of -1 stands for unlimited.
 
+The following fields are optional in case of broker running on local machine
+but are mandatory when using broker on remote computer. When there is a need to override the default values these fields are specified in application.conf file as additional `brokerConfig` parameters.
+
+`username` – login of the user connecting to remote machine, defaults to “guest”
+
+`password` – string of characters for user authentication, defaults to “guest”
+
+As was mentioned above the default values may be used only for the connection via local host.
```
Comment (rkatsyuryna) [inline doc/MSB.md:336]: Looks broker specific info need to be placed in amqp adapter section

File: doc/MSB.md [hunk 2]
```diff
@@ -326,6 +326,17 @@ See for more [detail](https://www.rabbitmq.com/tutorials/amqp-concepts.html).
...
+More references on how to configure the broker to allow the remote access with the default values “guest” can be found [here](https://www.rabbitmq.com/access-control.html).
```
Comment (rkatsyuryna) [inline doc/MSB.md:337]: Looks broker specific info need to be placed in amqp adapter section

File: doc/MSB.md (hunk 1 above)
Comment (ghost) [inline doc/MSB.md:336]: As these fields are specified for chosen broker and adapter I suggest to rename this configuration section into : Description of AMQP connection configuration fields

File: doc/MSB.md (hunk 2 above)
Comment (ghost) [inline doc/MSB.md:337]: The same as with the above

Comment (rkatsyuryna) [inline doc/MSB.md:337]: Yes. Sounds OK for me

Comment (ghost) [inline doc/MSB.md:337]: renamed the section, please merge if there is no other comments

**Pull request merged**